houston/errors.go

```diff
@@ -1,6 +1,27 @@
 package houston
 
-import "fmt"
+import (
+	"fmt"
+	"strings"
+)
+
+type ErrFieldsNotAvailable struct {
+	BaseError error
+}
+
+func (e ErrFieldsNotAvailable) Error() string {
+	return "Some fields requested by the CLI are not available in the server schema."
```

Comment:
nitpicking: should we log the actual error received from Houston as well, to be able to root cause the issue, since this log will only be present in case the verbosity is set to debug.

Reply:
Just above, the `ErrFieldsNotAvailable` error struct has a property `BaseError`.

The houston package should not be responsibel of logging errors, this is the responsibility of the package calling houston methods.

So in this case, I added the `BaseError`, so that in the version package I can do an error casting and logrus.Debug the base error so that the user is still able to read the houston error.